### PR TITLE
Support live streaming

### PR DESCRIPTION
**Pull request by diegomarcon**

### CODE REVIEW

File: Pod/Classes/Base/MediaControl.swift [hunk 1]
```diff
@@ -143,7 +165,25 @@ public class MediaControl: UIBaseObject {
...
+        updateBars()
+        updateScrubberPosition()
+        updatePlaybackControlButtonIcon()
+    }
+    
+    private func setupForLive() {
+        seekPercentage = 1
+        progressBarView.backgroundColor = UIColor.redColor()
+        durationLabel.text = ""
+        currentTimeLabel.text = ""
```
Comment (barbosa) [inline Pod/Classes/Base/MediaControl.swift]: Hey @diegomarcon,

there is a tiny white separator between the `durationLabel` and the `currentTimeLabel`. Do you hide it when dealing with live media?

File: Pod/Classes/Base/MediaControl.swift [hunk 2]
```diff
@@ -217,7 +263,7 @@ public class MediaControl: UIBaseObject {
         hideControlsTimer.invalidate()
     }
     
-    @IBAction func handleScrubberPan(panGesture: UIPanGestureRecognizer) {
+    @IBAction func handleScrubberPan(panGesture: UIPanGestureRecognizer) {
```
Comment (barbosa) [inline Pod/Classes/Base/MediaControl.swift]: There is an extra `tab` here.

File: Pod/Classes/Base/MediaControl.swift (hunk 1 above)
Comment (diegomarcon) [inline Pod/Classes/Base/MediaControl.swift]: @barbosa 

Actually I was not hiding it. It's a discreet separator but it does look odd when paying attention

**Comment by r-a-o** [comment]
Awesome! Just want to know if there is a way to provide multiple stream urls so player can pick based on network bandwidth?
Is this available in current implementation for live and on-demand videos? If its not currently present, I am willing to implement it, any pointer on this would help immensely. 

Thanks!

**Pull request merged**